src/main/bsAggregator.ts

```diff
@@ -1,11 +1,25 @@
-import { BSAggregator } from '@cityofzion/blockchain-service'
+import { Account, BSAggregator } from '@cityofzion/blockchain-service'
 import { exposeApiToRenderer } from '@cityofzion/bs-electron/dist/main'
 import { BSEthereum } from '@cityofzion/bs-ethereum'
 import { BSNeoLegacy } from '@cityofzion/bs-neo-legacy'
 import { BSNeo3 } from '@cityofzion/bs-neo3'
 import { TBlockchainServiceKey } from '@shared/@types/blockchain'
+import { SharedAccountHelper } from '@shared/helpers/SharedAccountHelper'
 
-import { getHardwareWalletTransport } from './hardwareWallet'
+export const getHardwareWalletTransport = async ({ address, blockchain }: Account<TBlockchainServiceKey>) => {
+  // This import is only to avoid circular dependencies
+  const { HardwareWalletGeneric } = await import('./hardwareWallet')
```

Comment:
I like this, but I hope it doesn't take too long to import.

Reply:
It's the only way to avoid circular dependency. I tried the same solution I use in NWM, but here I can't pass the service as a parameter, since IPC only accepts serializable values.